doc/python/getting-started.md

```diff
@@ -80,6 +80,19 @@ fig = go.Figure(data=go.Bar(y=[2, 3, 1]))
 fig.write_html('first_figure.html', auto_open=True)
 ```
 
+### Plotly chart in Dash
+
+[Dash](https://plotly.com/dash/) is the best way to build analytical apps in Python using Plotly figures. To run the app below, run `pip install dash`, click "Download" to get the code and run `python app.py`.
+
+Get started  with [the official Dash docs](https://dash.plotly.com/installation) and **learn how to effortlessly [style](https://plotly.com/dash/design-kit/) & [deploy](https://plotly.com/dash/app-manager/) apps like this with <a class="plotly-red" href="https://plotly.com/dash/">Dash Enterprise</a>.**
+
+
+```python hide_code=true
+from IPython.display import IFrame
+snippet_url = 'https://dash-gallery.plotly.host/python-docs-dash-snippets/'
+IFrame(snippet_url + 'getting-started', width='100%', height=630)
```

Comment:
Please use some nicer colors on this one and reply to this comment when done. There are some nice named sets here https://github.com/plotly/plotly.py/issues/2192

Reply:
I just pushed the changes, should be reflected in a few mins.